docker/dockerfiles/Dockerfile.ember-build

```
@@ -0,0 +1,25 @@
+FROM pix/nodebase-ember:latest as dev 
+
+
+
+WORKDIR /code
+COPY . . 
+
+
+RUN npm ci
+
+CMD [ "npm", "run", "start:watch" ]
+
+
+FROM dev as builder
+ENV BUILD_ENV=dev
+
+RUN ember build --env $BUILD_ENV
+
+
+
+FROM nginx:latest as production
```

```suggestion
FROM nginx:1.21.6 as production
```

C'est la version de Scalingo : https://github.com/Scalingo/nginx-buildpack/blob/master/config/versions.sh